src/react/components/DealOverview.tsx

```diff
@@ -65,11 +66,14 @@ export const DealOverview = () => {
 		switch (repaymentSelectValue) {
 			case "interest": {
 				const interest = calculateInterest();
+				const repayAmountFee = interest * 0.10; // 10 percent fee
 				setRepaymentAmount(interest);
+				setRepaymentAmountFee(repayAmountFee);
```

Comment:
we don't do anything with it besides displaying it in a notification. maybe inside the template string, see what kind of repayment it is and just display the value there without adding extra state stuff